code/modules/mob/living/carbon/human/life/handle_breath.dm

```diff
@@ -112,9 +112,12 @@
 					adjust_fire_stacks(0.5)
 					IgniteMob()
 		else
+			var/obj/item/clothing/suit/space/plasmaman/PS=wear_suit
```

Comment:
wear_suit may not specifically be /obj/item/clothing/suit/space/plasmaman should it pass to the else statement, be careful of this.

Reply:
uh, the else is for the !on_fire.. why is it showing as not for the on fire?

Reply:
oh wait my apologies; no no this is the already existing code for plasmamen as can be seen further down. As it stands, plasmamen can ONLY wear plasmamen suits or it'll runtime anyway, but yeah fair point I can fix that